### PR TITLE
Ben/active case linebreak

### PR DESCRIPTION
**Pull request by bionara**
# Description

Quick fix for line breaking number on homepage: added `numberOfLines` to the Text wrapper, and re-jigged the layout slightly to be more uniform. Screenshot has ",123" added to the number manually to demo the bug doesn't occur. We could make the font smaller if we exceed these numbers, but if that happens, we probably have bigger things to worry about than a line break un the UI...!!

https://www.notion.so/joinzoe/Fix-Wrapping-of-Active-Cases-e5cd0ef52b3e43b7a5bf731ec162e2a6

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

image

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if relevant)
- [ ] Added new screen(s) to `src/features/screens.ts` and new Redux slice(s) to `mocks/mockedInitialState.ts` (if relevant)

## Checklist (for reviewers)

- [ ] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Out of scope and potential follow-up

[comment]: <> (Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?)

### CODE REVIEW

File: src/components/cards/estimated-case/EstimatedCaseCard.tsx [hunk 1]
```diff
@@ -44,15 +44,15 @@ export function EstimatedCaseCard({
         </View>
       </View>
       <View style={styles.metricRow}>
-        <View style={styles.metricBox}>
-          <Text textClass="h1Regular">{leftMetric}</Text>
+        <View >
```
Comment (aikewoody) [inline src/components/cards/estimated-case/EstimatedCaseCard.tsx:47]: Needs a `yarn prettier` :)

File: src/components/typography/text/index.tsx [hunk 2]
```diff
@@ -32,6 +32,7 @@ export interface IProps {
   textAlign?: TTextAlign | undefined;
   textDecorationLine?: TTtextDecorationLine | undefined;
   textClass?: TTextClass;
+  numberOfLines?: number | undefined;
```
Comment (aikewoody) [inline src/components/typography/text/index.tsx:35]: It's probably a copy paste from above but this should def be `numberOfLines?: number;`

Comment (bionara) [reply src/components/typography/text/index.tsx:35]: updated

File: src/components/cards/estimated-case/EstimatedCaseCard.tsx (hunk 1 above)
Comment (bionara) [reply src/components/cards/estimated-case/EstimatedCaseCard.tsx]: prettiness ++

**Review by aikewoody — APPROVED** [review]

**Comment by selvach** [comment]
> We could make the font smaller if we exceed these numbers, but if that happens, we probably have bigger things to worry about than a line break un the UI...!!

😂

**Pull request merged**